Production/govuk_ios/ViewControllers/SearchViewController.swift

```diff
@@ -0,0 +1,131 @@
+import UIKit
+import UIComponents
+
+class SearchViewController: BaseViewController,
+                            TrackableScreen {
+    private let viewModel: SearchViewModel
+
+    private lazy var searchBar: UISearchBar = {
+        let localSearchBar = UISearchBar()
+
+        localSearchBar.searchTextField.backgroundColor = UIColor.govUK.fills.surfaceSearchBox
+        localSearchBar.enablesReturnKeyAutomatically = false
+        localSearchBar.translatesAutoresizingMaskIntoConstraints = false
+        localSearchBar.placeholder = "Search"
+        localSearchBar.backgroundImage = UIImage()
+        localSearchBar.searchTextField.attributedPlaceholder = NSAttributedString(
+            string: "Search",
+            attributes: [
+                NSAttributedString.Key.foregroundColor: UIColor.govUK.text.secondary,
+                NSAttributedString.Key.font: UIFont.govUK.body
+            ]
+        )
+        localSearchBar.searchTextField.leftView?.tintColor = UIColor.govUK.text.secondary
+        localSearchBar.searchTextField.addTarget(
+            self,
+            action: #selector(searchReturn),
+            for: UIControl.Event.editingDidEndOnExit
+        )
+
+        return localSearchBar
+    }()
+
+    var trackingName: String { "Search_Modal" }
+    var trackingTitle: String? { "Search" }
```

Comment:
Shouldn't need this because:
https://github.com/alphagov/govuk-mobile-ios-app/blob/develop/Production/govuk_ios/Model/Analytics/TrackableScreen.swift#L25